src/Manifest/Manifest.router.ts

```diff
@@ -36,6 +37,15 @@ export class ManifestRouter extends Router {
       server.handleRequest(this.getProjectManifest)
     )
 
+    /**
+     * Returns the manifest of a land coordinates associated project  (Builder In World)
+     */
+    this.router.get(
+      '/manifests',
+      withAuthentication,
```

Comment:
**Question** with this impelementation, an authenticated user can ask for other users's manifests. Is this correct? the endpoint below, has [withProjectAuthorization](https://github.com/decentraland/builder-server/pull/197/files#diff-536f13249592afdba56fe4ccd5ba79296b564dcc392bbd5049f46509ece9e6e0L73) which checks that the authorised user owns the project

Reply:
But it will work if we don't have the ID? I am looking at the code but it checks for ID, in my case, there won't be an ID, since I am looking for the new field.

Should I add to the `withProjectAuthorization `to check for any properties that are in the request?